### PR TITLE
Fix #214: Simplefind recognizes maxResults

### LINKED ISSUE #214: Simple Find does not seem to respect range
.../entity?R=from:0&maxResults:0&P=*:0

{"status":"COMPLETE","modifiedCount":0,"matchCount":19,"processed":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}],"hostname":"myhost"}

### PR DESCRIPTION
**Pull request by bserdar**

### CODE REVIEW

File: crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java [hunk 1]
```diff
@@ -309,8 +309,9 @@ public Response simpleFind(@PathParam(PARAM_ENTITY) String entity,
                                @QueryParam("P") String p,
                                @QueryParam("S") String s,
                                @QueryParam("from") Long from,
-                               @QueryParam("to") Long to) throws IOException {
-        return simpleFind(entity, null, q, p, s, from, to);
+                               @QueryParam("to") Long to,
+                               @QueryParam("maxResults") Long maxResults) throws IOException {
+        return simpleFind(entity, null, q, p, s, from, to, maxResults);
```
Comment (BVulaj) [inline crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java]: This list seems to be growing.  Does it make sense to replace this list with `@BeanParam`?

https://jersey.java.net/apidocs/latest/jersey/javax/ws/rs/BeanParam.html

Comment (bserdar) [inline crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java]: I'll look

Comment (bserdar) [inline crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java]: rest project is using jaxrs 1_1 spec, and that doesn't have beanparam

File: crud/src/test/java/com/redhat/lightblue/rest/crud/ITCaseCrudResourceTest.java [hunk 2]
```diff
@@ -288,7 +288,7 @@ public void testFirstIntegrationTest() throws IOException, ClassNotFoundExceptio
                 "name:1r,iso3code:1,iso2code:0r",
                 "name:a,iso3code:d,iso2code:d",
                 0l,
-                100l).getEntity().toString();
+                100l,null).getEntity().toString();
```
Comment (BVulaj) [inline crud/src/test/java/com/redhat/lightblue/rest/crud/ITCaseCrudResourceTest.java]: Can we add a test that tests for when maxResults != null?

Comment (bserdar) [inline crud/src/test/java/com/redhat/lightblue/rest/crud/ITCaseCrudResourceTest.java]: Added
